internal/flipflop/handler.go

```diff
@@ -132,11 +132,18 @@ func (cth *ConditionTaskHandler) Run(ctx context.Context) error {
 	case rctypes.PowerCycleBMC:
 		return cth.powerCycleBMC(ctx)
 	case rctypes.SetPowerState:
-		return cth.setPowerState(ctx)
+		return cth.setPowerState(ctx, cth.task.Parameters.ActionParameter)
 	case rctypes.SetNextBootDevice:
-		return cth.setNextBootDevice(ctx)
+		return cth.setNextBootDevice(
```

Comment:
Here as well. Why pass the task params into setNetBootDevice is cth is already passed into it?